ui/components/component-library/button-link/button-link.scss

```diff
@@ -0,0 +1,43 @@
+.mm-button-link {
+  color: var(--color-primary-default);
```

Comment:
Could we possibly store the opacity values for hover and disabled in css variables at the top of the file?